cmd/operator-sdk/bundle/build.go

```diff
@@ -0,0 +1,81 @@
+package bundle
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dirBuildArgs       string
+	tagBuildArgs       string
+	imageBuilderArgs   string
+	packageNameArgs    string
+	channelsArgs       string
+	channelDefaultArgs string
+	overwriteArgs      bool
+)
+
+// newBundleBuildCmd returns a command that will build operator bundle image.
+func newBundleBuildCmd() *cobra.Command {
+	bundleBuildCmd := &cobra.Command{
+		Use:   "build",
+		Short: "Build operator bundle image",
+		Long: `The opm bundle build command will generate operator
+        bundle metadata if needed and build bundle image with operator manifest
+        and metadata.
+
+        For example: The command will generate annotations.yaml metadata plus
+        Dockerfile for bundle image and then build a container image from
```

Comment:
If the user would not use OLM, then I do not think that these annotations should be added.  I understand that the OLM is optional for SDK users and not of them are using or would like to use it.